suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -293,13 +294,25 @@ const discoverAccountsByDescriptorThunk = createThunk(
                 if (accountInfo.empty) {
                     isFinalRound = true;
                 }
+                const isCoinEnablingActive = selectIsFeatureFlagEnabled(
+                    getState(),
+                    FeatureFlag.IsCoinEnablingActive,
+                );
+
+                const isVisible =
+                    // If the account is not empty, it should be visible.
+                    !accountInfo.empty ||
+                    // If the feature flag is enabled, first normal account should be visible.
+                    (isCoinEnablingActive &&
+                        bundleItem.accountType === 'normal' &&
```

Comment:
Why only normal type and why only 0 index? Shoudn't it be last account instead of first?

Reply:
@Nodonisko 
First empty normal account because there might be more invisible accounts and we want to show user the first possible and not to skip any. If user had empty account on index0 and visible on index 1, he could have received there funds and this would break discovery causing index 1 not to be discovered since we stop on first empty account.

Normal account because we dont want to show empty account for for example ledger or legacy types only this default(normal) is visible if empty for enabled coins.

This logic is similar to what desktop does.